docs/guides/algorithmiq-tem.ipynb

```diff
@@ -165,70 +167,145 @@
-    "\n",
-    "You can fine-tune your calculations and get more verbose information by passing additional options to TEM."
+    "You can return the results as:"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 7,
-   "id": "c2ce65e3",
+   "execution_count": null,
+   "id": "ad5b9a53",
    "metadata": {},
    "outputs": [],
    "source": [
-    "import logging\n",
+    "result = job.result()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e5e138d3",
+   "metadata": {},
+   "source": [
+    "<Admonition type=\"info\">\n",
+    "  The expected value for the noiseless circuit for the given operator should be around `0.18409094298943401`.\n",
+    "</Admonition>\n",
     "\n",
-    "options = {\n",
-    "        \"backend_name\": backend_name,\n",
-    "        \"num_max_shots\": 1_000,\n",
-    "        \"simulate_with_noise_model\": True,\n",
-    "        \"mitigate_readout_error\": True,\n",
-    "        \"num_readout_calibration_shots\": 10_000,\n",
-    "        \"logging_level\": logging.WARNING\n",
-    "    }\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c6ffb685",
+   "metadata": {},
+   "source": [
+    "## Inputs\n",
+    "\n",
+    "**Parameters**\n",
+    "Name | Type | Description | Required | Default | Example\n",
+    "-- | -- | -- | -- | -- | --\n",
+    "`pubs` | Iterable[EstimatorPubLike] | An iterable of PUB-like (primitive unified bloc) objects, such as tuples `(circuit, observables)` or `(circuit, observables, parameters, precision)`. See [Overview of PUBs](/guides/primitive-input-output#overview-of-pubs) for more information. The circuits don't need to be ISA circuits. | Yes | N/A | (circuit, observables)\n",
+    "`instance` | str | The hub/group/project to use in that format. | Yes | N/A |\"hub1/group1/project1\"\n",
+    "`backend_name` | str | Name of the backend to make the query.| No | If not provided, the least busy backend will be used. | \"ibm_fez\"\n",
+    "`options` | dict | Input options. See `Options` section for more details. | No | See `Options` section for more details.| \\{\"max_bond_dimension\": 100\\}\n",
+    "\n",
+    "\n",
+    "<Admonition type=\"caution\"> TEM does not currently support parametrized circuits. The `parameters` argument should be set to `None` if `precision` is specified.</Admonition>\n",
+    "\n",
+    "<Admonition type=\"caution\"> TEM currently supports circuits with linear connectivity. This restriction will be removed in future versions.</Admonition>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1ebd5f9b",
+   "metadata": {},
+   "source": [
+    "\n",
+    "### Options\n",
+    "\n",
+    "A dictionary containing the advanced options for the TEM. The dictionary may contain the keys in the following table. If any of the options are not provided, the default value listed in the table will be used. The default values are good for typical use of TEM.\n",
+    "\n",
+    "Name | Choices | Description  | Default\n",
```

Comment:
The Options table doesn't render correctly for me (the other tables do). It could just be me though.

Reply:
This is what it looks like on my end, Jessie - 
<img width="889" alt="Screenshot 2024-10-15 at 9 56 04 AM" src="https://github.com/user-attachments/assets/aca9b018-5a29-428b-8087-698f36903850">